src/applications/personalization/dashboard-2/components/CTALink.jsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
+import useLastWord from '../useLastWord';
+
 const CTALink = ({ ariaLabel, href, text, onClick, newTab }) => {
   const relProp = newTab ? 'noreferrer noopener' : undefined;
   const targetProp = newTab ? '_blank' : undefined;
 
+  const [lastWord, firstWords] = useLastWord(text);
```

Comment:
Definitely open to changing the name of this hook. But it returns an array of two strings:
1. the last word of the passed-in string
2. everything leading up to the last word